CHANGELOG.md

```diff
@@ -4,6 +4,40 @@ Date format: (year/month/day)
 
 ## Change Log
 
+### V4.6.8 (2019/11/04)
```

Comment:
todo add:

https://github.com/NLog/NLog/pull/3662 and https://github.com/NLog/NLog/pull/3660